### PR TITLE
entangled: testing cclient via cmake.

### PR DESCRIPTION
**Pull request by oopsmonk**

### CODE REVIEW

File: entangled-prs/pipeline.yml [hunk 1]
```diff
@@ -216,6 +216,20 @@ steps:
           - .:/workdir
         shell: ["/bin/bash", "-e", "-c"]
 
+  - name: "CMake - CClient"
+    command: cd /workdir && mkdir build && cd $_ && cmake .. && make -j16 && make -j16 test
```
Comment (thibault-martinez) [inline entangled-prs/pipeline.yml]: `mkdir build` may fail on some platform due to the `BUILD` file already existing.
```
thibault@MacBook-Pro-de-Thibault ~/i/entangled> mkdir build
mkdir: build: File exists
```

File: entangled/pipeline.yml [hunk 2]
```diff
@@ -217,6 +217,20 @@ steps:
           - .:/workdir
         shell: ["/bin/bash", "-e", "-c"]
 
+  - name: "CMake - CClient"
+    command: cd /workdir && mkdir build && cd $_ && cmake .. && make -j16 && make -j16 test
```
Comment (thibault-martinez) [inline entangled/pipeline.yml]: Ditto

File: entangled-prs/pipeline.yml (hunk 1 above)
Comment (oopsmonk) [reply entangled-prs/pipeline.yml]: The CI container is Ubuntu 19.10.

Comment (thibault-martinez) [reply entangled-prs/pipeline.yml]: Yes ignore me, I guess I was tired when I wrote that...

**Pull request merged**